arcgis-ios-sdk-samples/Content Display Logic/Storyboard/Main.storyboard

```diff
@@ -78,13 +78,33 @@
                                     <action selector="closeAction" destination="HRz-Eg-7ly" eventType="touchUpInside" id="JSz-9v-XS9"/>
                                 </connections>
                             </button>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="erh-VF-k0q">
```

Comment:
These buttons seem a little small. They have a point size of 15, but I think the standard size is 17. If you create a new button, it actually doesn't have a font at all (and presumably uses the default system font and size). I couldn't find a way to remove the font from a button (so that it goes back to having no font), but you could re-create the buttons to achieve that.